_mobile/android.md

```diff
@@ -124,13 +124,14 @@ for (int i = 0; i < scores.length; i++) {
 String className = ImageNetClasses.IMAGENET_CLASSES[maxScoreIdx];
 ```
  
-In the following sections you can find detailed explanation of pytorch android api, code walk through for bigger [demo application](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp), implementation details of api and how to customize and build it from the source. 
+In the following sections you can find detailed explanation of pytorch android api, code walk through for bigger [demo application](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp), 
```

Comment:
Line 111: "It's" --> "Its"

Line 127: "...detailed explanation[s] of [PyTorch Android API], code walk-through for [a] bigger demo application..."